### ISSUE TITLE
Importing from Jetlog with negative durations displays negative duration in Airtrail

### ISSUE THREAD

**Issue by nirenjan** [johanohly/AirTrail#108]
### The bug

I imported several flights from Jetlog into Airtrail, mainly because it's easier to import manually into Jetlog, then use the jetlog export to import into Airtrail. However, for a couple of flights, the jetlog export has a negative `duration`. because the flight departs and arrives at on the same date, but at an earlier local time. Once imported, the bulk of the flights that have a positive duration in the import file seem to be show the duration just fine, but for these flights with a negative duration, Airtrail shows a flight duration of -27h -30m.

However, I can edit these flights, and save without actually editing anything, and the flight duration is correctly recalculated, so it's not really a big deal.

### Operating System that is running AirTrail

Ubuntu

### AirTrail Version

1.1.0

### Your docker-compose.yml content

```YAML
services:
  db:
    container_name: airtrail_db
    image: postgres:16-alpine
    restart: always
    env_file:
      - .env
    environment:
      POSTGRES_DB: ${DB_DATABASE_NAME}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_PASSWORD: ${DB_PASSWORD}
    volumes:
      - db_data:/var/lib/postgresql/data
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U ${DB_USERNAME} -d ${DB_DATABASE_NAME}"]
      interval: 5s
      timeout: 5s
      retries: 5

  airtrail:
    container_name: airtrail
    image: johly/airtrail:latest
    restart: always
    env_file:
      - .env
    ports:
      - 1380:3000
    depends_on:
      db:
        condition: service_healthy

volumes:
  db_data:
```

### Your .env content

```Shell
# Your domain, e.g https://example.com
# You might have to add :443 if you are using https through a reverse proxy
ORIGIN=https://XXXXXXXX

# The database URL used by the application.
# If you are using the provided docker-compose file, you should only change the PASSWORD_HERE
# If you are using your own database, you should change this to the correct URL
DB_URL=postgres://airtrail:XXXXXXXX@db:5432/airtrail

# Values below this line are only for the default provided postgres database
###################################################################################
# Connection secret for postgres. You should change it to a random password
# Please use only the characters `A-Za-z0-9`, without special characters or spaces
# When you change the DB_PASSWORD, you should also update the DB_URL accordingly
DB_PASSWORD=XXXXXXXX

# The values below this line do not need to be changed
###################################################################################
DB_DATABASE_NAME=airtrail
DB_USERNAME=airtrail
```

### Reproduction steps

1. Import Jetlog import with the CSV contents below: 
[negative-duration.csv](https://github.com/user-attachments/files/17711362/negative-duration.csv)

2. Observe negative flight duration as shown in below screenshot: 
![Screenshot from 2024-11-11 21-06-10](https://github.com/user-attachments/assets/80f7349a-9c0a-46dc-9195-f3308ecd5696)

3. Edit flights and resave without modifying any data.
4. Observe corrected flight duration as shown below: 
![Screenshot from 2024-11-11 21-06-32](https://github.com/user-attachments/assets/ace80784-e31a-4cf3-96ba-4e1eb1218037)

**Comment by nirenjan** [comment]
Looking back at my original data that I had imported into Jetlog, the flights concerned did not have a duration field attached to them, and therefore, Jetlog had computed the duration based on its assumption that all timestamps were in the same timezone. This obviously results in incorrect duration values in the export, and I can't really blame Airtrail's import system for parsing garbage data.

However, I would ask if you would consider ignoring the duration field from Jetlog and instead, use the departure and arrival, failing that, to estimate the flight time.

**Comment by johanohly** [comment]
Good find, there is indeed no reason to use the duration provided by JetLog, as theirs is at best equal to AirTrail's calculation, as they (I assume) don't handle the arrival time being before the departure time very well. 
This will be available in the next release, although I am not sure when that will be. 

Thank you again for reporting this!

**Comment by johanohly** [comment]
Released in v1.1.1
